src/PlaywrightSharp.Abstractions/IElementHandle.cs

```diff
@@ -15,14 +16,56 @@ public interface IElementHandle : IJSHandle
         /// <returns>
         /// A <see cref="Task"/> that completes when the screenshot is done, yielding the screenshot as a <see cref="T:byte[]" />.
         /// </returns>
-        Task<byte[]> ScreenshotAsync(ScreenshotOptions options = null);
-
+        Task<byte[]> ScreenshotAsync(ScreenshotOptions options = null);
+
+        /// <summary>
+        /// focuses the element and triggers an `input` event after filling.
+        /// If element is not a text `&lt;input&gt;`, `&lt;textarea&gt;` or `[contenteditable]` element, the method throws an error.
+        /// </summary>
+        /// <param name="text">Value to set for the `&lt;input&gt;`, `&lt;textarea&gt;` or `[contenteditable]` element.</param>
+        /// <returns>A <see cref="Task"/> that completes when the fill action is done.</returns>
+        Task FillAsync(string text);
+
         /// <summary>
         /// Content frame for element handles referencing iframe nodes, or null otherwise.
         /// </summary>
         /// <returns>A <see cref="Task"/> that completes when the frame is resolved, yielding element's parent <see cref="IFrame" />.</returns>
-        Task<IFrame> ContentFrameAsync();
-
+        Task<IFrame> ContentFrameAsync();
+
+        /// <summary>
+        /// Scrolls element into view if needed, and then uses <see cref="IPage.Mouse"/> to hover over the center of the element.
+        /// </summary>
+        /// <returns>A <see cref="Task"/> that completes when the element is successfully hovered.</returns>
+        Task HoverAsync();
+
+        /// <summary>
+        /// Tries to scroll element into view, unless it is completely visible as defined by <see href="https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API"/>'s <b>ratio</b>.
+        /// </summary>
+        /// <seealso cref="IElementHandle.VisibleRatioAsync"/>
+        /// <returns>A <see cref="Task"/> that completes when the element is successfully scrolled into view.</returns>
+        Task ScrollIntoViewIfNeededAsync();
+
+        /// <summary>
+        /// Returns the frame containing the given element.
+        /// </summary>
+        /// <returns>A <see cref="Task"/> that completes when the frame is resolved, yielding element's owner <see cref="IFrame" />.</returns>
+        Task<IFrame> OwnerFrameAsync();

```

Comment:
```suggestion
        Task<IFrame> GetOwnerFrameAsync();

```

Reply:
I'll create a PR reviewing this. But let's give these kinds of methods a C# flavor prefixing it with `Get`.